tasks.py

```diff
@@ -614,6 +626,28 @@ def vpc_components(ctx):
     return subnet_id, sg_id
 
 
+def oc_base_url(ctx):
+    cmd = ("aws {} opsworks --region us-east-1 describe-stacks "
+           "--query \"Stacks[?Name=='{}'].StackId\" --output text")\
+        .format(profile_arg(), OC_CLUSTER_NAME)
+    res = ctx.run(cmd, hide=1)
+    stack_id = res.stdout.strip()
+
+    cmd = ("aws {} opsworks describe-layers --stack-id {} "
+           "--query \"Layers[?Name=='Admin'].LayerId\" --output text")\
+        .format(profile_arg(), stack_id)
+    res = ctx.run(cmd, hide=1)
+    admin_layer_id = res.stdout.strip()
+
+    cmd = ("aws {} opsworks describe-instances --stack-id {} "
+           "--query \"Instances[?LayerIds[0]=='{}'].PublicDns\" --output text")\
```

Comment:
I don't think `PublicDns` is an attribute in the opsworks describe-instances output?